From: Debian Python Team <team+python@tracker.debian.org>
Date: Sat, 4 Nov 2023 10:34:21 +0200
Subject: Use the python3 binary to execute tests

Upstream runs the tests inside tox, which has a python binary available.
We don't currently use this, because their tox config isn't shipped in
their source tarball.

Forwarded: not-needed

--- a/test/prog
+++ b/test/prog
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # PYTHON_ARGCOMPLETE_OK
 """Test script used by test.TestBash."""
 
--- a/test/test.py
+++ b/test/test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import argparse
 import contextlib
 import os
@@ -1381,7 +1381,7 @@
             self.sh.run_command("export PATH=$PATH:./bin")
             self.sh.run_command("export PYTHONPATH=.:$PYTHONPATH")
             test_package = os.path.join(TEST_DIR, "test_package")
-            command = "pip install {} --target .".format(test_package)
+            command = "pip3 install {} --target .".format(test_package)
             if not wheel:
                 command += " --no-binary :all:"
             install_output = self.sh.run_command(command)
